### PR TITLE
[Won't pass CI] Updating the runtime framework versions for 1.0.5 and 1.1.1

### PR DESCRIPTION
**Pull request by livarcocc**
Updating the runtime framework versions for 1.0.5 and 1.1.1 and updating the CLI branding to 1.0.4.

@eerhardt @dotnet/dotnet-cli

### CODE REVIEW

File: build.proj [hunk 1]
```diff
@@ -19,12 +19,12 @@
...
     <CoreSetupChannel>release/1.1.0</CoreSetupChannel>
     <SharedFrameworkName>Microsoft.NETCore.App</SharedFrameworkName>
-    <SharedFrameworkVersion>1.1.1</SharedFrameworkVersion>
+    <SharedFrameworkVersion>1.1.2</SharedFrameworkVersion>
     <SharedHostVersion>1.1.0</SharedHostVersion>
     <HostFxrVersion>1.1.0</HostFxrVersion>
 
-    <CoreCLRVersion>1.1.1</CoreCLRVersion>
-    <JitVersion>1.1.1</JitVersion>
+    <CoreCLRVersion>1.1.2</CoreCLRVersion>
```
Comment (eerhardt) [inline build.proj]: @gkhanna79 - did CoreCLR and the JIT packages get versioned?

**Comment by gkhanna79** [comment]
Yes they are always updated with each release.


Thanks,
Gaurav
________________________________

File: branchinfo.txt [hunk 2]
```diff
@@ -3,7 +3,7 @@
 # Each line is expected to be in the format "[Name]=[Value]".
 MAJOR_VERSION=1
 MINOR_VERSION=0
-PATCH_VERSION=3
+PATCH_VERSION=4
 RELEASE_SUFFIX=rc4
```
Comment (eerhardt) [inline branchinfo.txt]: Is this correct?

Comment (livarcocc) [inline branchinfo.txt]: Yep, that's the next one. 1.0.3 is the latest out there. What are your concerns? I may be missing something.

Comment (eerhardt) [inline branchinfo.txt]: `RELEASE_SUFFIX=rc4`

Comment (livarcocc) [inline branchinfo.txt]: That can stay as is. it shows up in the version, but when we release we don't use the suffix at all. I don't see a reason to change it.

**Pull request merged**